src/InvokePlaster.ps1

```diff
@@ -24,10 +24,13 @@ Please follow the scripting style of this file when adding new script.
     General notes
 #>
 function Invoke-Plaster {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSAvoidUsingWriteHost', '', Scope='Function')]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSAvoidShouldContinueWithoutForce', '', Scope='Function', Target='CopyFileWithConflictDetection')]
```

Comment:
As long as we are using 1.8.0 in the AppVeyor build, we should be able to remove these.  Let me take a crack at it.